libc/src/__support/UInt.h

```diff
@@ -43,13 +43,20 @@ struct BigInt {
   static_assert(is_integral_v<WordType> && is_unsigned_v<WordType>,
                 "WordType must be unsigned integer.");
 
+  using word_type = WordType;
```

Comment:
> Will making these type aliases word_t, unsigned_t, and signed_t conflict with anything? 

It won't AFAICT.
In the `cpp` namespace we adopt the STL format so I sticked to the usual container's `value_type`.
Also TBH I would rather use `element_type` here instead of `word_type` but that would mean more changes.